src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -86,8 +42,12 @@ for i = 6, #arg do
   end
   headers[#headers + 1] = parts[#parts - 1]..'/'..parts[#parts]
 
+  if full_path:sub(-12) ~= '.generated.h' then
+    goto continue
```

Comment:
This means lua 5.1 can't be used to build. Did we already have that requirement?